From 6c11f64fde274f186017e9634dc259c7ffe213f2 Mon Sep 17 00:00:00 2001
From: RiccardoPaniz <92019843+RiccardoPaniz@users.noreply.github.com>
Date: Tue, 9 Jul 2024 16:58:02 +0200
Subject: [PATCH] Create README.md

---
 README.md | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..9daeafb
--- /dev/null
+++ b/README.md
@@ -0,0 +1 @@
+test
-- 
2.30.2.windows.1

